src/main/java/com/agh/surveys/model/group/dto/GroupDto.java

```diff
@@ -0,0 +1,26 @@
+package com.agh.surveys.model.group.dto;
+
+import com.agh.surveys.model.group.Group;
+import com.agh.surveys.model.user.User;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+@NoArgsConstructor
+public class GroupDto {
```

Comment:
Proponuje trochę inną konwencje nazywania klas DTO, dla GroupDto to w sumie nie wiadomo o co chodzi. 
Możemy założyć, że takie klasy powinny mieć informacje co one robię, typu: Transfer/Create/Remove/Update:Group